corehq/apps/app_manager/tests/test_util.py

```diff
@@ -31,6 +32,27 @@ def test_advanced_form_get_action_type(self):
         self.assertEqual(modules[0]['forms'][0]['action_type'], 'load (load_0)')
 
 
+class TestGetDefaultFollowupForm(SimpleTestCase):
+    def test_default_followup_form(self):
+        app = Application.new_app('domain', "Untitled Application")
+
+        parent_module = app.add_module(AdvancedModule.new_module('parent', None))
```

Comment:
nitpick: I'd use `Module` instead of `AdvancedModule` here because you're testing basic functionality, not functionality specific to advanced modules